--- setup.py.orig	2012-07-15 23:15:24.000000000 +0400
+++ setup.py	2012-10-09 16:28:05.413440311 +0400
@@ -596,9 +596,17 @@
         'console_scripts': [
             'ptdump = tables.scripts.ptdump:main',
             'ptrepack = tables.scripts.ptrepack:main',
-            'nctoh5 = tables.netcdf3.scripts.nctoh5:main [netCDF]',
             ],
         }
+    # Create nctoh5 script only when ScientificPython is installed:
+    try:
+        import Scientific.IO.NetCDF
+    except ImportError:
+        pass
+    else:
+        setuptools_kwargs['entry_points']['console_scripts'].append(
+            'nctoh5 = tables.netcdf3.scripts.nctoh5:main [netCDF]'
+            )
     # Test suites.
     setuptools_kwargs['test_suite'] = 'tables.tests.test_all.suite'
     setuptools_kwargs['scripts'] = []
@@ -616,7 +624,15 @@
         'tables.tests', 'tables.nodes.tests',
         'tables.netcdf3.tests', 'tables.nra.tests']
     setuptools_kwargs['scripts'] = [
-        'utils/ptdump', 'utils/ptrepack', 'utils/nctoh5']
+        'utils/ptdump', 'utils/ptrepack']
+    # Create nctoh5 script only when ScientificPython is installed:
+    try:
+        import Scientific.IO.NetCDF
+    except ImportError:
+        pass
+    else:
+        setuptools_kwargs['scripts'].append('utils/nctoh5')
+        
 # Copy additional data for packages that need it.
 setuptools_kwargs['package_data'] = {
     'tables.tests': ['*.h5'],
